test/config/PayeConfigSpec.scala

```diff
@@ -65,6 +65,22 @@ class PayeConfigSpec extends UnitSpec with MockitoSugar {
       config.scottishTaxBandKeys shouldBe expected
     }
 
+    "retrieve UK tax band keys in order for a valid year" in {
+      val config = new PayeConfig {
+        override protected val configPath: String = "paye_config.conf"
```

Comment:
Why don't you just use paye.conf here? Is paye_config.conf just for testing? If it is, this test doesn't test anything in production code.

Reply:
Agreed
deleted the paye_config.conf and now using paye.conf instead for testing